fog-libvirt.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency("fog-core", ">= 1.27.4")
   s.add_dependency("fog-json")
-  s.add_dependency("fog-xml", "~> 0.1.1")
+  s.add_dependency("fog-xml", "~> 0.1")
```

Comment:
This allows all versions up to `1.x` if I am not mistaken including `0.1` and `0.2`. `~>` operator allows to iterate on the last specified number (expect when this is a single number).

Reply:
Hmm, I think you are right. So I guess this PR is kinda pointless, then.